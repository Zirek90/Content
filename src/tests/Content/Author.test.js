import React from "react";
import renderer from "react-test-renderer";
import AuthorComponent from "../../components/Content/Author";

describe("AuthorComponent", () => {
  it("renders correctly", () => {
    const props = "John Doe";
    const tree = renderer.create(<AuthorComponent author={props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
