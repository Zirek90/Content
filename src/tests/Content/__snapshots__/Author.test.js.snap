// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorComponent renders correctly 1`] = `
<p
  className="sc-gtssRu iBDuNi"
>
  John Doe
</p>
`;
